include/galsim/fmath/fmath.hpp

```diff
@@ -33,6 +33,11 @@
 #include <stdlib.h>
 #include <float.h>
 #include <string.h> // for memcpy
+
+#ifdef __x86_64__
```

Comment:
I think this is probably too restrictive.  For instance, I think `__i386__` works just fine with fmath.
Can we use `__SSE__` as a proxy for when fmath is expected to work?  That's the fundamental feature it needs, right?